.github/workflows/cache_data.yaml

```
@@ -19,7 +19,7 @@ jobs:
     steps:
       # Checkout current git repository
       - name: Checkout
-        uses: actions/checkout@v2.4.0
+        uses: actions/checkout@v3
```

Perhaps better to pin explicitly to v3.0.0? What do you think @seisman?
```suggestion
        uses: actions/checkout@v3.0.0
```